doc/source/configuration/cloudkitty.rst

```diff
@@ -0,0 +1,140 @@
+==========
+CloudKitty
+==========
+
+Configuring in kayobe-config
+============================
+
```

Comment:
I generally find it helpful to link to other relevant docs. You could link to the k-a cloudkitty config docs (using :kolla-ansible-doc:)

Reply:
Done :)